src/Persistence/ObjectManager.php

```diff
@@ -4,6 +4,9 @@
 
 namespace Doctrine\Persistence;
 
+use Doctrine\DBAL\Exception;
+use Doctrine\ORM\Exception\ORMException;
+use Doctrine\ORM\ORMInvalidArgumentException;
```

Comment:
You can reference the DBAL or the ORM here; you would have to use an ancestor that is common to all three types, and known by `doctrine/persistence`.

Reply:
so I only use one type of Exception instead of 3 ?